docs/pt/docs/external-links.md

```diff
@@ -0,0 +1,123 @@
+# Links externos e Artigos
+
+**FastAPI** tem uma grande comunidade em crescimento constante.
+
+Existem muitas postagens, artigos, ferramentas e projetos relacionados ao **FastAPI**.
+
+Aqui tem uma lista, incompleta, de algumas delas.
+
+!!! tip
```

Comment:
Serrones, aqui não seria melhor também colocar em português? 
```suggestion
!!! dica
```

Reply:
Unfortunately we can't. This world adds a text block, and if we translate, it would probably break.